pandas/io/gbq.py

```diff
@@ -682,6 +686,13 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
 
         .. versionadded:: 0.19.0
 
+    **kwargs: Arbitrary keyword arguments
```

Comment:
this is just like using a named argument. what I would like to see is something like:

```
config = {'query' : ....}
read_gbq(.....,configuration=config)
```
then just add these keys directly.